src/test/java/cart/ProductIntegrationTest.java

```diff
@@ -1,37 +1,89 @@
 package cart;
 
-import io.restassured.RestAssured;
+import static io.restassured.RestAssured.*;
+import static org.assertj.core.api.Assertions.*;
+
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.web.server.LocalServerPort;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 
-import static io.restassured.RestAssured.given;
-import static org.assertj.core.api.Assertions.assertThat;
+import cart.controller.request.ProductCreateRequest;
+import cart.controller.request.ProductUpdateRequest;
+import cart.repository.ProductRepository;
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class ProductIntegrationTest {
 
-    @LocalServerPort
-    private int port;
+	@Autowired
+	ProductRepository repository;
+	@LocalServerPort
+	private int port;
+
+	@BeforeEach
+	void setUp() {
+		RestAssured.port = port;
+	}
+
+	@Test
+	public void getProducts() {
+		final ExtractableResponse<Response> result = given()
+			.contentType(MediaType.APPLICATION_JSON_VALUE)
+			.when()
+			.get("/products")
+			.then()
```

Comment:
log all을 통해 요청 응답을 로깅하는 건 어떤가요? 테스트는 기능 명세의 의미도 있어 요청 응답을 볼 수 있으면 도움이 됩니다.
```suggestion
			.when().log().all()
			.get("/products")
			.then().log().all()
```

Reply:
log all을 하니 이렇게 한눈에 알아볼 수 있는 로그가 찍히네요! 감사합니다><

```java
Body:
{
    "name": "KIARA",
    "price": 10000.0,
    "image": "이미지"
}
HTTP/1.1 201 
Location: /products/3
Content-Length: 0
Date: Sun, 30 Apr 2023 14:51:21 GMT
Keep-Alive: timeout=60
Connection: keep-alive
```

Reply:
오 리뷰 중 길게 남긴게 있는데요, (https://github.com/woowacourse/jwp-shopping-cart/pull/229#discussion_r1181518781) log all을 통해 어떤 요청이 어떻게 직렬화가 되고 어떤 응답이 어떻게 역직렬화가 되는지 눈으로 확인할 수 있으니, ResponseEntity와 Request, Response 객체를 이리저리 건드려보며 요청 ~ 응답 까지의 흐름에 대해 감을 잡아 보시면 서버의 동작 과정에 대해 좋은 이해가 생길거 같네요 ㅎ

Reply:
넵 감사합니다! 삽질이 많이 필요해보여요 허헣